camel_test.go

```diff
@@ -24,6 +24,7 @@ func TestUpperCamelCase(t *testing.T) {
 		"skip   many spaces":    "SkipManySpaces",
 		"skip---many-dashes":    "SkipManyDashes",
 		"skip___many_underline": "SkipManyUnderline",
+		"XRequestId":            "XRequestId",
```

Comment:
Please add another test with more than one capital letter:
```suggestion
		"XRequestId":            "XRequestId",
		"HTTPStatusCode":        "HttpStatusCode",
```